.github/workflows/main.yml

```diff
@@ -14,9 +14,10 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Setup JDK 1.8
-        uses: actions/setup-java@v1
+        uses: actions/setup-java@v2
         with:
-          java-version: 1.8
+          java-version: 8
+          distribution: 'zulu'
```

Comment:
what's zulu about?

Reply:
See square/moshi#1375.

Reply:
haha so really it's: "because @JakeWharton said so"

fine by me, I don't have any spare brain cycles to think on this.

However: please don't resolve conversations after answering. That's not for you to do, it's for the reviewer to decide if something is resolved.

Reply:
roger that.